awswrangler/s3/_read_parquet.py

```diff
@@ -189,9 +166,10 @@ def _read_parquet_metadata(
     boto3_session: boto3.Session,
     s3_additional_kwargs: Optional[Dict[str, str]],
     version_id: Optional[Union[str, Dict[str, str]]] = None,
-    pyarrow_additional_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
I'm slightly concerned we're removing these escape hatches. Is this because we can't use them as much in distributed scenario?

Reply:
In the specific case of `read_parquet_metadata`, if you follow along how `pyarrow_additional_kwargs` is used, you should see that it's used to [supply](https://github.com/awslabs/aws-data-wrangler/blob/main/awswrangler/s3/_read_parquet.py#L43) `coerce_int96_time_unit` and nothing else. So I thought we might as well be specific with this one as `pyarrow_additional_kwargs` gives the false impression that you can supply more

For all other methods, you should be able to supply anything you wish via `arrow_additional_kwargs`

Reply:
Cool agreed